eng/pipelines/azure-pipelines-public.yml

```diff
@@ -64,7 +64,7 @@ variables:
     value: ''
 
   # Use the build reason to decide whether to include helix_tests
-  - ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+  - ${{ if or(notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.DefinitionName'], 'azdo-tests')) }}:
```

Comment:
```suggestion
  - ${{ if or(notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.Reason'], 'Manual')) }}:
```

Reply:
If the reason is not PullRequest then the tests already run. But this is not the case for `/azp run` in a PR comment.